metrics/core/metrics.go

```diff
@@ -755,32 +758,121 @@ var MetricAcceleratorMemoryTotal = Metric{
 	MetricDescriptor: MetricDescriptor{
 		Name:        "accelerator/memory_total",
 		Description: "Total accelerator memory (in bytes)",
-		Labels:      acceleratorLabels,
```

Comment:
Keep this.

Reply:
Will do.

Reply:
Minor nit: if you move this field from last to here, the diff would be shorter.

Reply:
Will do.